apps/browser-extension-wallet/.env.defaults

```
@@ -62,6 +62,7 @@ POSTHOG_HOST=https://e.lw.iog.io
 POSTHOG_EXCLUDED_EVENTS=
 # set this variable to true only in release packages. By having this set to false, we ensure that we will not be using Post Hog production projects tokens in development stage
 PRODUCTION_MODE_TRACKING=false
+POSTHOG_FEATURE_FLAG_CHECK_FREQUENCY_SECONDS=5
```

I think that's too frequent. Maybe consult with product to understand what's the max delay we can tolerate for pushing feature flags
```suggestion
POSTHOG_FEATURE_FLAG_CHECK_FREQUENCY_SECONDS=600
```